Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.yml

```diff
@@ -0,0 +1,47 @@
+category: Forensics & Malware Analysis
+commonfields:
+  id: WildFire-Reports
+  version: -1
+configuration:
+- defaultvalue: https://wildfire.paloaltonetworks.com/publicapi
+  display: Server base URL (e.g., https://192.168.0.1/publicapi)
+  name: server
+  required: true
+  type: 0
+- display: API Key
+  name: token
+  required: true
```

Comment:
Change to `required: false`.
If the user leaves this empty (this will be the most common case), use `demisto.getLicenseCustomField("WildFire-Reports.token")`
This will fetch the API key from the license (if it is present).